usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -577,3 +586,91 @@ def fetch_naics_hierarchy(naics: str) -> dict:
         pass
     results = {"toptier_code": toptier_code, "midtier_code": midtier_code, "base_code": base_code}
     return results
+
+
+def fetch_account_details_award(award_id: int) -> dict:
+    queryset = (
+        FinancialAccountsByAwards.objects.filter(award_id=award_id)
+        .values("disaster_emergency_fund")
+        .annotate(
+            obligated_amount=Sum("obligations_delivered_orders_unpaid_total_cpe"),
+            gross_outlay_amount=Sum("gross_outlay_amount_by_award_cpe"),
```

Comment:
This is a tricky field that needs special handling beyond FY 2020. See slack thread: https://data-transparency-bfs.slack.com/archives/C0122HUCD17/p1591885771087400

and **CARES Award Obligation and Awards Outlay … by Budget Category (File C):**
section in https://federal-spending-transparency.atlassian.net/browse/DEV-5360 which addresses this.

IF we absolutely **MUST** defer the work that makes this work post-2020, put a comment in with `ENABLE_CARES_ACT_FEATURES` in the comment that states that we need to further accommodate this problem post-launch.

From thread:
> If you want to get ALL-TIME (life of the award) value, it’s trickier, and you need to look at all FYs that had this award, and get the latest reported period’s value in any of those FYs. e.g. Let’s say we’re in FY21, Q2, period 6 (period 5 has closed):
> 1. If we only need to look back at 2020, get FY2020 P12 value for this award. 
> 2. Add that to FY2021 P05 value for this award
> 3. That gives the lifetime outlay value for this award over the 2 FYs it existed.

There's a rough attempt at what sql would look like that grabs that cross-year values.